WordPress/Classes/ViewRelated/Me/My Profile/Gravatar/GravatarInfoRow.swift

```diff
@@ -0,0 +1,103 @@
+import Foundation
+import WordPressShared
+import DesignSystem
+
+struct GravatarInfoRow: ImmuTableRow {
+    var action: ImmuTableAction?
+
+    func configureCell(_ cell: UITableViewCell) {
+        guard let cell = cell as? GravatarInfoCell else { return }
+        cell.applyColors()
+        cell.selectionStyle = .none
+    }
+
+    static let cell = ImmuTableCell.class(GravatarInfoCell.self)
+}
+
+class GravatarInfoCell: WPTableViewCellDefault {
+
+    private enum Constants {
+        static let externalLinkLogo: UIImage? = UIImage(named: "icon-post-actionbar-view")?.withRenderingMode(.alwaysTemplate)
+        static let infoText = NSLocalizedString("Gravatar keeps your profile information safe and up to date, automatically syncing any updates made here with your Gravatar profile.", comment: "This text is shown in the profile editing page to let the user know about Gravatar.")
+        static let linkText = NSLocalizedString("Learn more on Gravatar.com", comment: "This is a link that takes the user to the external Gravatar website")
+        static var font: UIFont { TextStyle.caption.uiFont }
+        static let gravatarLink = "https://gravatar.com"
+    }
+
+    private var infoLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = Constants.font
+        label.text = Constants.infoText
+        label.adjustsFontForContentSizeCategory = true
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private var linkTextView: UITextView = {
+        let text = UITextView()
+        text.translatesAutoresizingMaskIntoConstraints = false
+        text.isUserInteractionEnabled = true
```

Comment:
Nit: I think the default value of `isUserInteractionEnabled` is true so this line might be redundant.

Reply:
Correct. This was a label so apparently I forgot to remove that after turning it to UITextView. Done now.